spec/requests/content_item_requests/message_bus_spec.rb

```diff
@@ -141,4 +141,73 @@
       end
     end
   end
+
+  context "/v2/publish" do
+    let(:request_body) { { update_type: "major" }.to_json }
+    let(:request_path) { "/v2/content/#{content_id}/publish" }
+    let(:request_method) { :post }
+
+    before do
+      draft = FactoryGirl.create(:draft_content_item, v2_content_item)
+      FactoryGirl.create(:version, target: draft, number: 1)
+    end
+
+    it "sends a message with the 'format.update_type' routing key" do
+      Timecop.freeze do
+        do_request
+
+        expect(response.status).to eq(200)
+
+        expected_payload = v2_content_item.except(:access_limited).merge(
+          update_type: "major",
+          transmitted_at: DateTime.now.to_s(:nanoseconds),
+        ).to_json
+
+        delivery_info, _, payload = wait_for_message_on(@queue)
+
+        expect(delivery_info.routing_key).to eq("guide.major")
+        expect(JSON.parse(payload)).to eq(JSON.parse(expected_payload))
+      end
+    end
+  end
+
+  describe "content type inlining" do
```

Comment:
Stripping all HTML would be a problem, because we are quite likely to want to be able to use things like <h1> tags to mark some parts of the text as more important.  Plus, some fields may contain identifiers with characters like < which are significant.